.github/workflows/netlify.yaml

```diff
@@ -0,0 +1,53 @@
+on:
+  pull_request:
+    branches: [main, master]
+
+name: pkgdown
+
+jobs:
+  netlify:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-tinytex@v2
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::pkgdown, local::.
+          needs: website
+
+      - name: Install package
+        run: R CMD INSTALL .
+
+      - name: Create website
+        run: |
+          pkgdown::build_site()
+        shell: Rscript {0}
+
+      - name: Create index file
+        run: |
+          echo '<!DOCTYPE html><html><head><meta http-equiv="refresh" content="0;URL=/dev/index.html" />  <script language="javascript"> window.location.replace('/dev/index.html')</script></head></html>' > ./docs/index.html
+
+      - name: Deploy to Netlify
+        uses: nwtgck/actions-netlify@v3.0
+        with:
+          publish-dir: './docs'
+          production-branch: main
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          deploy-message:
+            'Deploy from GHA: ${{ github.event.pull_request.title || github.event.head_commit.message }} (${{ github.sha }})'
+          # these default to 'true'
+          enable-commit-comment: false
+          enable-github-deployment: false
```

Comment:
I think this is the only change that's only significant — I don't know the netlify action started doing this by default, but we don't need it and GHA don't have permission for it by default.